test/test-shutdown.py

```diff
@@ -21,7 +21,9 @@ def run(args):
             "TERM": "linux",
         }, encoding='utf-8', timeout=60)
 
-    if args.verbose:
+    if args.logfile:
+        console.logfile = open(args.logfile,'w')
```

Comment:
The log file should be shown somewhere if the test fails, otherwise it makes potential failures in CI harder to debug.

Reply:
is this test run by CI ? If so can you show me where ? Thanks.

Reply:
- https://jenkins-systemd.apps.ocp.cloud.ci.centos.org/job/upstream-vagrant-archlinux/10524/artifact//systemd-centos-ci/artifacts_wni922al/vagrant-logs.vww/vagrant-arch-testsuite.TXm/TEST-69-SHUTDOWN_PASS.log
- https://jenkins-systemd.apps.ocp.cloud.ci.centos.org/job/upstream-centos9s/325/artifact//systemd-centos-ci/artifacts_w9zp_4mg/testsuite-logs-upstream.p7r/TEST-69-SHUTDOWN_1_PASS.log

Reply:
Thanks, I added an info message that should show the path of the log file.

Reply:
This won't help us in CI - it needs to be dumped either to journal or on stdout/stderr, as these are the only artifacts that are exported.

Reply:
I see. In that case it might make more sense to use the log file only when the test is run manually (ie outside of CI).

Do we have a way to detect this case ?

Reply:
I'm not aware of any. We have `$INTERACTIVE_DEBUG`, but that wouldn't help (much) here.

Reply:
ok then I'll drop the unconditional use of the new `--logfile` option but will leave the option around so people can use it when debugging manually.

Would that be ok for you ?

Reply:
Yup, that sound good, thank you.

Reply:
Great, new version submitted.